Cross.toml

```diff
@@ -4,5 +4,5 @@ dockerfile = "./docker/linux-cross/Dockerfile"
 [build.env]
 passthrough = [
     "AWS_LC_SYS_INTERNAL_NO_PREFIX", "AWS_LC_RUST_INTERNAL_BINDGEN", "AWS_LC_FIPS_SYS_INTERNAL_NO_PREFIX", "GOPROXY",
-    "AWS_LC_FIPS_SYS_STATIC", "AWS_LC_SYS_STATIC"
+    "AWS_LC_FIPS_SYS_STATIC", "AWS_LC_SYS_STATIC", "AWS_LC_SYS_CMAKE_BUILDER", "AWS_LC_SYS_CC_SRC_COLLECTOR"
```

Comment:
I originally had expected to need to pass additional build options to CMake in order to have the build generate sufficient debug information from which we could identify all of the source files used.  However, it's turned out to not be needed so far.  I'd like to leave it listed here in case its needed when we expand the set of platforms supported by this mechanism.